### PR TITLE
bpo-29455: Mention coverage.py in trace module documentation

### PR DESCRIPTION
**Pull request by marco-buttu**
This pull request fixes [bpo-29455](https://bugs.python.org/issue29455).

### CODE REVIEW

File: Doc/library/trace.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ annotated statement coverage listings, print caller/callee relationships and
 list functions executed during a program run.  It can be used in another program
 or from the command line.
 
+.. seealso:: `Coverage.py <https://coverage.readthedocs.io/>`_, which is one
```
Comment (Carreau) [inline Doc/library/trace.rst]: Looking at the rest of the documentation is seem like most `.. seealso::` are alone on their lines with the paragraph on the next one.

Comment (marco-buttu) [reply Doc/library/trace.rst]: Thanks @Carreau. I also changed the markup to [reST list](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-seealso).

File: Doc/library/trace.rst [hunk 2]
```diff
@@ -13,6 +13,12 @@ annotated statement coverage listings, print caller/callee relationships and
 list functions executed during a program run.  It can be used in another program
 or from the command line.
 
+.. seealso::
+
+   `Coverage.py <https://coverage.readthedocs.io/>`_
+      A popular third-party coverage tools that provides very nice HTML
```
Comment (berkerpeksag) [inline Doc/library/trace.rst]: tools -> tool?

Comment (berkerpeksag) [reply Doc/library/trace.rst]: "very nice" can be dropped.

File: Doc/library/trace.rst [hunk 3]
```diff
@@ -13,6 +13,12 @@ annotated statement coverage listings, print caller/callee relationships and
...
+      output along with advanced features such as branch coverage.
```
Comment (berkerpeksag) [inline Doc/library/trace.rst]: advanced -> more advanced?

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

**Review by berkerpeksag — APPROVED** [review]
LGTM, thanks! The issue is assigned to @brettcannon so I won't merge this.

**Pull request merged**